CHANGELOG.adoc

```diff
@@ -2,15 +2,27 @@
 
 This file documents all notable changes to https://github.com/devonfw/sonar-devon-plugin[sonar-devon-plugin].
 
-== Unreleased 3.1.0
+== 3.2.0
+
+This version marks the second official release. Next to the publication of this plugin on the SonarQube Marketplace, multiple new rules were added:
+
+* Checks for adherence to the Devon naming convention
+* Check for the correct mapping of of custom datatypes
+* Check for security constraint annotations on use-case implementation methods
+
+Also, issues raised by our rules will now be of type Code Smell and tagged with 'architecture-violation'.
+
+Changes for this release can be found in https://github.com/devonfw/sonar-devon-plugin/milestone/3?closed=1[milestone 3.2.0].
+
+== 3.1.0
 
 Changes for this release can be found in https://github.com/devonfw/sonar-devon-plugin/milestone/2?closed=1[milestone 3.1.0].
 
 == 3.0.0
 
 This is the first official release. In http://devonfw.com[devon] all released artifacts share the same version number.
 This allows to easily see which artifacts belong together and are guaranteed to be compatible. As this plugin was later
-added to devon it started with the devon version for that moment in time what was 3.0.0.
+added to devon, it started with the devon version for that moment in time what was 3.0.0.
```

Comment:
Can you replace `devon` (the entire word, not in `devon4j`) with `devonfw`. Thanks.

Reply:
Otherwise ready to merge. I am already approving but if possible you can update this as well.